src/test/java/uk/gov/pay/directdebit/payments/services/GoCardlessServiceOnDemandTest.java

```diff
@@ -30,42 +35,56 @@ public void setUp() {
     }
 
     @Test
-    public void confirm_shouldStoreAGoCardlessCustomerBankAccountMandateAndPaymentWhenReceivingConfirmTransaction() {
+    public void confirm_shouldCreateAGoCardlessCustomerBankAccountMandateAndPaymentWhenReceivingConfirmTransaction() {
         Mandate mandate = mandateFixture.toEntity();
+        var confirmMandateResponse = 
+                new PaymentProviderConfirmMandateResponse(
+                        SandboxMandateId.valueOf("aPaymentProviderId"),
+                        MandateBankStatementReference.valueOf(BANK_ACCOUNT_ID));
 
-        when(mockedGoCardlessClientFacade.createMandate(mandate, goCardlessCustomer)).thenReturn(goCardlessMandate);
+        when(mockedGoCardlessClientFacade.createMandate(mandate, goCardlessCustomer)).thenReturn(confirmMandateResponse);
```

Comment:
Can probably just inline the creation of `confirmMandateResponse` here.

Reply:
👍